### ISSUE TITLE
Systemd service prevents KDE Wayland from autologin

### ISSUE THREAD

**Issue by Vitorvlv** [LizardByte/Sunshine#1134]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Issue is mentioned in the comments of #901, just creating a proper issue for it.

 I get stuck in the login screen, manually login in works, but sunshine has not started. If I try to start the service after login in then it works fine.

### Host Operating System

Linux

### Operating System Version

Fedora 37

### Architecture

64 bit

### Sunshine commit or version

0.19.1

### Package

Linux - rpm

### GPU Type

AMD

### GPU Model

5700XT

### GPU Driver/Mesa Version

22.3.7

### Config

```Shell
not relevant, fresh install
```

### Relevant log output

```shell
This is what the systemd logs show after manually login in.

xdg-desktop-autostart.target: Found dependency on graphical-session.target/start
xdg-desktop-autostart.target: Found dependency on sunshine.service/start
xdg-desktop-autostart.target: Found dependency on xdg-desktop-autostart.target/start
```

**Comment by ReenigneArcher** [comment]
Is this issue present in v0.18.4?

**Comment by Vitorvlv** [comment]
Yes

**Comment by Vitorvlv** [comment]
At least for me seems like the issue solved itself, the new systemd service does seems to work as intended.

Sunshine started to not even open at all, always segfaulting, after a few hours troubleshooting I realized it started to happen after I specified a (detected) monitor in the Audio/Video part of the settings. Removed that from the settings file and sunshine started to open again, after that I also began noticing that the service was behaving just fine across reboots. So, maybe the segfault was interfering with it, maybe it was someone else, there was an update today. 

Either way, everything's working as intended (apart from specifying an output causing a segfault), so I'm closing this. Thanks!!!
